src/ansys/systemcoupling/core/client/grpc_client.py

```diff
@@ -156,7 +156,7 @@ def start_container_and_connect(
         LOG.debug("...started")
         self._connect(_LOCALHOST_IP, port)
 
-    def start_pim_and_connect(self, start_output: bool, version: str = None):
+    def start_pim_and_connect(self, start_output: bool = False, version: str = None):
```

Comment:
Sorry @SeunSomefun - this should have occurred to me but it might be better to put the new argument at the end. It's a minor point but it means that if there are any scripts that rely on argument order rather than named arguments, they will continue to work.

Reply:
No worries, I've made the change!